<launch>
	<arg name="marker_size" default="2.65" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic_bottom" default="/naoqi_driver_node/camera/bottom/image_raw" />
	<arg name="cam_info_topic_bottom" default="/naoqi_driver_node/camera/bottom/camera_info" />
	<arg name="cam_image_topic_front" default="/naoqi_driver_node/camera/front/image_raw" />
	<arg name="cam_info_topic_front" default="/naoqi_driver_node/camera/front/camera_info" />
	<!-- <arg name="output_frame" default="/base_link" /> -->
	<arg name="output_frame" default="/base_footprint" />

	<node name="ar_track_alvar_bottom" pkg="ar_track_alvar" type="individualMarkersForChess" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic_bottom) $(arg cam_info_topic_bottom) $(arg output_frame)" />

	<node name="ar_track_alvar_front" pkg="ar_track_alvar" type="individualMarkersForChess" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic_front) $(arg cam_info_topic_front) $(arg output_frame)" />
</launch>
